src/data/matrix/basic.lean

```diff
@@ -7,6 +7,7 @@ import algebra.big_operators.pi
 import algebra.module.pi
 import algebra.big_operators.ring
 import data.fintype.card
+import algebra.char_p
```

Comment:
If we don't find a better place, it could just go in the file where I use it, or in maybe `linear_algebra/matrix`. Alternately, is there some general pi-instance approach that we could put in `algebra/char_p`?

Reply:
I think `data/matrix/char_p.lean` is the best solution for now. It will be easy to move the contents to another place later, when needed.